bots/honkbot.py

```diff
@@ -205,21 +201,33 @@ def is_extended_maintenance_time():
     @commands.command()
     async def eamuse(self, ctx):
         """
-        Gets eAmusement maintenance time.
+        Gets e-amusement maintenance time.
 
-        Japanese Servers: 5:00 AM to 7:00 AM JST every day
+        Japanese Servers: 5:00 AM to 7:00 AM JST on weekdays
         Extended and US Servers: 3:00 AM to 7:00 AM JST on the third Tuesday
         """
 
         if self.is_extended_maintenance_time():
-            ddr_message = self.get_display_time("us")
-            other_message = self.get_display_time("extended")
+            await ctx.send(("**Extended Maintenance today. "
+                            "All games and e-amusement websites under maintenance from " +
+                            f"{self.get_display_time('extended')}. " +
+                            f"{self.messages['website_down']} " +
+                            f"{self.messages['card_down']}**"))
         else:
-            ddr_message = ":white_check_mark: - no maintenance today"
-            other_message = self.get_display_time("normal")
+            ddr_message = f"{self.messages['us_games']}: **No maintenance today.**"
+            website_message = ("**Website under maintenance from " +
```

Comment:
Some PEP8 nitpicking: https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

Reply:
Oh, this looks much better.

Wish my IDE would tell me about this one.